docs/source/release/v6.10.0/Framework/Data_Handling/Bugfixes/37151.rst

```diff
@@ -0,0 +1 @@
+- GenerateGroupingPowder now accepts `GroupingWorkspace` as an optional argument instead of required
```

Comment:
```rst
- :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` now accepts ``GroupingWorkspace`` as an optional argument instead of required
```